docs/source/introduction.rst

```diff
@@ -0,0 +1,27 @@
+Introduction
+~~~~~~~~~~~~
+
+|Software| is program to call, impute, and phase genotypes using a pedigree in potentially very large livestock populations. A complete description of the methods is given in Whalen et al. (2018; http://dx.doi.org/10.1186/s12711-018-0438-2).
```

Comment:
This is a repetition, but a good one - let's have this paragraph both here and on the `index.rst` so that people get info about software directly on the main documentation page - likely also in the README file (please check this - I think we have it - writing this on the fly).